src/pages/Player.vue

```diff
@@ -401,6 +482,34 @@ export default {
   },
   methods: {
     ...mapActions("auth", ["setAccessToken", "setActiveWorkspace"]),
+    calculateScorecardMetrics() {
+      // calculate scorecard metrics using `itemResponses`
+      if (this.itemResponses != undefined && this.items != undefined) {
+        this.numOfCorrect = 0;
+        this.numOfWrong = 0;
+        this.numOfSkipped = 0;
+        this.itemResponses.forEach((itemResponse, itemIndex) => {
+          const userAnswer = this.itemResponses[itemIndex].answer;
+          if (this.isItemMCQ(itemIndex)) {
+            const correctAnswer = this.items[itemIndex].details.correct_answer;
+            if (!isNaN(userAnswer)) {
+              if (userAnswer == correctAnswer) this.numOfCorrect += 1;
+              else this.numOfWrong += 1;
+            } else this.numOfSkipped += 1;
+          } else if (this.isItemSubjective(itemIndex)) {
+            if (userAnswer == null) this.numOfSkipped += 1;
```

Comment:
this seems like a redundant operation to loop through all questions even if just say one question is skipped. there should be some way of making it reactive.